packages/opal-common/opal_common/schemas/data.py

```diff
@@ -26,6 +25,15 @@ class DataSourceEntry(BaseModel):
         "PUT", description="Method used to write into OPA - PUT/PATCH"
     )
 
+class DataSourceEntryWithPollingInterval(DataSourceEntry):
```

Comment:
Do we really want to do some inheritance here ?
This New attribue should be directly add to the class DataSourceEntry.
Docs also need to be update accordingly.

Reply:
@loan75 I didn't want to add it to the DataSourceEntry base class because that would imply that you could set the Pulling Interval on requested data changes made by something like a data update REST call tot he server. This keeps pulling interval siloed as only a configuration var.

Reply:
@orweis I'm confused on what you're asking for in terms of what we are trying to toggle. The goal with this PR is to be able to do a partial or full rehydration of OPA services at a set interval.

Reply:
@orweis why wouldn't you just remove the Polling interval from the data source config JSON object you are passing.

Reply:
I guess that's a vlid point.
As well as your previous one, re versus the update.
😇

Reply:
@DServy good point, that make sense then to do that.
@orweis seem fair, like that it will be the same bahavior as policy_poller_interval, only editable when deploying/starting the application.